### PR TITLE
PTS site update with construction docs

### PR DESCRIPTION
**Pull request by phillipsm**
Refined version of PTS site. Includes fleshed out desktop site and an open site with the the freely licensed construction docs

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,204 +1,191 @@
...
+        <div class="col-md-5">
+          <img src="img/walker.png" class="img-fluid">
+        </div>
+        <div class="col-md-7">
+          <h2>Measured Benefits</h2>
+          <p>
+            Open working environments overexpose us. Too many sounds. Too many distractions. They force us to multi-task and <a href="https://www.inc.com/geoffrey-james/why-your-company-will-benefit-from-getting-rid-of-open-office-spaces-first-90.html" target="_blank">keep us</a> from being our best selves.
+          </p>
+          <p>
+            When we increase our privacy, our focus improves, and we are able to perform at significanly higher levels. <a href="https://www.newyorker.com/business/currency/the-open-office-trap" target="_blank">We become</a> more creative, our cognitive abilities sharpen, our stress levels decrease, our <a href="http://www.bbc.com/capital/story/20170105-open-offices-are-damaging-our-memories" target="_blank">sick days drop, and we add 15% to the bottom line</a>.
```
Comment (bensteinberg) [inline index.html]: "significanly" s/b "significantly"

File: index.html [hunk 2]
```diff
@@ -1,204 +1,191 @@
...
+          </p>
+          <p>
+            The benefits of working in a focused space <a href="http://amj.aom.org/content/23/1/101.short" target="_blank">are clear and long established</a>. And, thanks to the leightweight nature of the spaces, you can pop them up in an afternoon - instantly adding an environment that encourages focus, happiness, and productivity.
```
Comment (bensteinberg) [inline index.html]: "leightweight" s/b "lightweight"

**Review by bensteinberg — COMMENTED** [review]
Couple of typos

**Comment by phillipsm** [comment]
updated. thanks, @bensteinberg !!

**Pull request merged**